roles/mesos-agent/tasks/main.yml

```diff
@@ -12,7 +12,7 @@
     - { file: '/etc/mesos-slave/credential', content: '/etc/mesos-slave/.credentials' }
     - { file: '/etc/mesos-slave/fetcher_cache_size', content: "2GB" }
     - { file: '/etc/mesos-slave/fetcher_cache_dir', content: "/tmp/mesos" }
-    - { file: '/etc/mesos-slave/work_dir', content: "/mnt/mesos" }
+    - { file: '/etc/mesos-slave/work_dir', content: "/log/mesos" }
```

Comment:
Maybe use a toplevel `/mesos` instead? Unless other apps will log there. Then agent logs will be in `/mesos/slaves`.